cmd/vela-server/main.go

```diff
@@ -146,6 +146,12 @@ func main() {
 			Name:    "vault-token",
 			Usage:   "vault token for storing secrets",
 		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_SECRET_VAULT_VERSION", "SECRET_VAULT_VERSION"},
+			Name:    "vault-version",
+			Usage:   "vault k/v backend version to utilize",
+			Value:   "1",
```

Comment:
Objections to bumping to this to 2 by default? Seems like the majority of users would be on k/v v2 instead of v1.

Reply:
I don't have any problems with that 👍 

However, if we do set v2 as the default, then could we also make sure the compose Docker image for Vault is updated?

https://github.com/go-vela/server/blob/e13901e1efab8c19b2e5ee302e65a9228de798e9/docker-compose.yml#L62-L74

Reply:
Updated to default to v2 and utilize the `vault:latest` image within the `docker-compose.yml` file.